relay-server/src/metrics_extraction/conditional_tagging.rs

```diff
@@ -2,7 +2,11 @@ use relay_dynamic_config::TaggingRule;
 use relay_general::protocol::Event;
 use relay_metrics::Metric;
 
-pub fn run_conditional_tagging(event: &Event, config: &[TaggingRule], metrics: &mut [Metric]) {
+pub fn run_conditional_tagging<'a>(
+    event: &Event,
+    config: impl Iterator<Item = &'a TaggingRule>,
```

Comment:
changed to an iterator because I wanted to merge the global and project tagging rules without any new allocations, lmk if this is overkill